src/lib/multicall/executeMulticall.ts

```diff
@@ -361,3 +359,18 @@ function getRequest(callEntries: [string, { callData: MulticallFetcherConfig[num
 
   return requests;
 }
+
+function isOldIOS(threshold: number) {
```

Comment:
maybe let's remove threshold argument? we need to detect specific version because of quite specific error. 

and i think it makes sense to add some comment that says that there is an issue with serialization/deserialization on iOS <= 16

Reply:
and 16 could be moved to a constant